src/main/resources/templates/languages/en/non-strategic/rptSouthernWeeklyHearingList.json

```diff
@@ -0,0 +1,15 @@
+{
+  "listName": "Residential Property Tribunal: Southern region Weekly Hearing List",
+  "contactMessage": "Members of the public wishing to observe a hearing or representatives of the media may, on their request, join any telephone or video hearing remotely while they are taking place by sending an email in advance to the tribunal at [insert office email] with the following details in the subject line “[OBSERVER/MEDIA] REQUEST – [case reference] – [hearing date]” and appropriate arrangements will be made to allow access where reasonably practicable.",
+  "tableHeaders": [
+    "Date",
+    "Time",
+    "Venue",
+    "Case Type",
```

Comment:
Can we keep the casing consistent. Case type instead of Case Type

Reply:
done.